scripts/normalize-json

```diff
@@ -17,7 +18,11 @@ else:
 	fd_in.close()
 	fd_out=open(sys.argv[1],'w')
 
-fd_out.write(json.dumps(data, indent=2, sort_keys=True))
+str = json.dumps(data, indent=2, sort_keys=True)
+str = re.sub(r'\s+$', '', str, 0, re.M)
+str = re.sub(r'\s+$', '', str, flags=re.M)
```

Comment:
Aren't these two lines identical? Why do we need both.
(also, I tend to recommend spelling out options like MULTILINE, for clarity)